main.html

```diff
@@ -1,47 +1,72 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-<meta charset="UTF-8">
-<title>Page Title</title>
-<link rel="stylesheet" href="main.css">
-<link rel="stylesheet" href="normalize.css">
+    <meta charset="UTF-8">
+    <meta content='width=device-width, initial-scale=1.0, user-scalable=0' name='viewport'/>
+    <title>Page Title</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/8.0.1/normalize.css">
+    <link rel="stylesheet" href="main.css">
 </head>
 <body>
     <header>
```

Comment:
<img width="449" alt="Screen Shot 2022-06-25 at 1 02 15 PM" src="https://user-images.githubusercontent.com/18077388/175788921-8b4d1e02-9330-4112-a741-98eda73a1a37.png">

doubled header, you can use only header class="header" here

Reply:
+